src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Startup.cs

```diff
@@ -73,38 +73,41 @@ public void ConfigureServices(IServiceCollection services)
                 .AddEntityFrameworkStores<ApplicationDbContext>();
 #elif (OrganizationalAuth)
 #if (GenerateApiOrGraph)
-            string[] scopes = Configuration.GetValue<string>("CalledApi:CalledApiScopes")?.Split(' ');
+            string[] initialScopes = Configuration.GetValue<string>("DownstreamApi:Scopes")?.Split(' ');
+
 #endif
+            services.AddAuthentication(OpenIdConnectDefaults.AuthenticationScheme)
+                    .AddMicrosoftIdentityWebApp(Configuration.GetSection("AzureAd"))
 #if (GenerateApiOrGraph)
-            services.AddMicrosoftWebAppAuthentication(Configuration, "AzureAd")
-                .AddMicrosoftWebAppCallsWebApi(Configuration, scopes, "AzureAd")
-                .AddInMemoryTokenCaches();
-#else
-            services.AddMicrosoftWebAppAuthentication(Configuration, "AzureAd");
-#endif
+                        .EnableTokenAcquisitionToCallDownstreamApi(initialScopes)
 #if (GenerateApi)
-            services.AddDownstreamWebApiService(Configuration);
+                            .AddDownstreamWebApi("DownstreamApi", Configuration.GetSection("DownstreamApi"))
 #endif
 #if (GenerateGraph)
-            services.AddMicrosoftGraph(scopes, Configuration.GetValue<string>("CalledApi:CalledApiUrl"));
+                            .AddMicrosoftGraph(Configuration.GetSection("DownstreamApi"))
+#endif
+                            .AddInMemoryTokenCaches();
+#else
+                    ;
```

Comment:
Having a separate semicolon on a new line is weird in our templates. Let's move away from this pattern. What we'd want is something like:

```C#
            services.AddAuthentication(OpenIdConnectDefaults.AuthenticationScheme)
#if (GenerateApiOrGraph)
                    .AddMicrosoftIdentityWebApp(Configuration.GetSection("AzureAd"))
                        .EnableTokenAcquisitionToCallDownstreamApi(initialScopes)
#if (GenerateApi)
                            .AddDownstreamWebApi("DownstreamApi", Configuration.GetSection("DownstreamApi"))
#endif
#if (GenerateGraph)
                            .AddMicrosoftGraph(Configuration.GetSection("DownstreamApi"))
#endif
                            .AddInMemoryTokenCaches();
#else
                    .AddMicrosoftIdentityWebApp(Configuration.GetSection("AzureAd"));
#endif
```

This also applies to a bunch of other places in this PR.

FYI I'm going to be much more picky about styles in this PR since it's now RC1 instead of previews.

Reply:
Done.